code/controllers/subsystem/ticker.dm

```diff
@@ -221,21 +221,21 @@ var/datum/subsystem/ticker/ticker
 	//initialise our cinematic screen object
 	cinematic = new /obj/screen{icon='icons/effects/station_explosion.dmi';icon_state="station_intact";layer=21;mouse_opacity=0;screen_loc="1,0";}(src)
 
-	var/obj/structure/bed/temp_buckle = new(src)
 	if(station_missed)
 		for(var/mob/M in mob_list)
-			M.buckled = temp_buckle				//buckles the mob so it can't do anything
+			M.notransform = TRUE //stop everything moving
 			if(M.client)
 				M.client.screen += cinematic	//show every client the cinematic
 	else	//nuke kills everyone on z-level 1 to prevent "hurr-durr I survived"
 		for(var/mob/M in mob_list)
-			M.buckled = temp_buckle
 			if(M.client)
 				M.client.screen += cinematic
 			if(M.stat != DEAD)
 				var/turf/T = get_turf(M)
 				if(T && T.z==1)
 					M.death(0) //no mercy
```

Comment:
not setting no transform on them as they are going to die, also this caused a tonne of runtimes because they were unbuckling from temp_buckle which wasn't actually even buckling correctly